### PR TITLE
llm: remove random bolding

### PR DESCRIPTION
**Pull request by MIfoodie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
N/A

### CODE REVIEW

File: pages/common/llm.md [hunk 1]
```diff
@@ -19,10 +19,10 @@
 
 `llm install {{package1 package2 ...}}`
 
-- Download and run a prompt against a [m]odel:
+- Download and run a prompt against a model:
```
Comment (fazlearefin) [inline pages/common/llm.md:22]:
```suggestion
- Download and run a prompt against a [model]:
```

File: pages/common/llm.md [hunk 2]
```diff
@@ -19,10 +19,10 @@
...
 
 `llm --model {{orca-mini-3b-gguf2-q4_0}} "{{What is the capital of France?}}"`
 
-- Have an interactive chat with a specific [m]odel:
+- Have an interactive chat with a specific model:
```
Comment (fazlearefin) [inline pages/common/llm.md:26]:
```suggestion
- Have an interactive chat with a specific [model]:
```

Comment (MIfoodie) [reply pages/common/llm.md:26]: Why would we even have the brackets in the first place?

Comment (fazlearefin) [reply pages/common/llm.md:26]: > Why would we even have the brackets in the first place?

Please see this section [Style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#short-option-mnemonics)

**Review by fazlearefin — APPROVED** [review]

File: pages/common/llm.md [hunk 3]
```diff
@@ -19,10 +19,10 @@
 
 `llm install {{package1 package2 ...}}`
 
-- Download and run a prompt against a [m]odel:
+- Download and run a prompt against a [model]:
```
Comment (fazlearefin) [inline pages/common/llm.md:22]:
```suggestion
- Download and run a prompt against a model:
```

I am sorry for the confusion caused earlier. I went through the [Style Guide Short Option Mnemonics](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#short-option-mnemonics] again; complete words options like `model` should not have any mnemonic at all. @kbdharun or @vitorhcl can clarify here.

File: pages/common/llm.md [hunk 4]
```diff
@@ -19,10 +19,10 @@
...
 
 `llm --model {{orca-mini-3b-gguf2-q4_0}} "{{What is the capital of France?}}"`
 
-- Have an interactive chat with a specific [m]odel:
+- Have an interactive chat with a specific [model]:
```
Comment (fazlearefin) [inline pages/common/llm.md:26]:
```suggestion
- Have an interactive chat with a specific model:
```

File: pages/common/llm.md (hunk 3 above)
Comment (kbdharun) [reply pages/common/llm.md:22]: Exactly, mnemonics only be used to highlight short options or their context.

**Comment by kbdharun** [comment]
Thanks for your contribution, unfortunately, we don't support enclosing full words as mnemonics.

**Pull request closed without merging**